src/functionalTest/java/uk/gov/hmcts/reform/blobrouter/FunctionalTestBase.java

```diff
@@ -31,7 +34,27 @@ protected void setUp() {
             config.sourceStorageAccountKey
         );
 
-        this.blobRouterStorageClient = new BlobServiceClientBuilder()
+        BlobServiceClientBuilder blobServiceClientBuilder = new BlobServiceClientBuilder();
+
+        if (config.useProxyForSourceStorage) {
+            blobServiceClientBuilder = blobServiceClientBuilder.httpClient(
+                new NettyAsyncHttpClientBuilder()
+                    .proxy(
+                        new ProxyOptions(
+                            ProxyOptions.Type.HTTP,
+                            new InetSocketAddress(
+                                "proxyout.reform.hmcts.net",
+                                8080
+                            )
+                        )
+                    )
+                    .build()
+            );
+
+            System.out.println("PROXY set for blobRouterStorageClient");
```

Comment:
can this be removed?

Reply:
I will but let me test it and see it in post func test.

Reply:
@mdayican As far as I understand this is supposed to be removed